flake.nix

```diff
@@ -53,5 +61,26 @@
           (containers "oci-images" {ci.publish = true;})
         ];
       };
+
+      std.harvest = {
+        packages = [["local" "packages"] ["local" "jobs"]];
+        devShells = [["local" "envs"] ["desktop" "envs"]];
+        hydraJobs = ["desktop" "hydraJobs"];
+      };
+
+      perSystem = {
+        pkgs,
+        system,
+        ...
+      }: {
+        _module.args.pkgs = import inputs.nixpkgs {
+          inherit system;
+          config = {
+            allowUnfree = true;
```

Comment:
Why do we need unfree packages?

Reply:
@gytis-ivaskevicius

Reply:
not sure, we probably can remove it. may have been copy paste mistake